src/components/controls/controls.scss

```diff
@@ -1,9 +1,11 @@
 $controls-column-max-width: 265px;
 $controls-column-compressed-max-width: 95px;
 $controls-transition: 0.3s;
+$toolbar-height: 4rem;
 
 .controls {
   z-index: 1;
+  padding-bottom: $toolbar-height;
```

Comment:
When I scrolled to the bottom, the controls or inspector accordions seemed cut off. Turned out they were cut off by the toolbar height.

Reply:
Could this be added to our `_settings.variables.scss` file?